base/shell/explorer/lang/bg-BG.rc

```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     Explorer for ReactOS
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Bulgarian Language resource file
+ * TRANSLATORS:
+ */
+
```

Comment:
Re. headers, I would rather do this myself in a separate commit, this kind of maintenance should be done by the core team.

Reply:
UPD: committed update for headers in master and resolved the conflict here now.